monero-wallet-gui.pro

```diff
@@ -116,6 +116,7 @@ LIBS += -L$$WALLET_ROOT/lib \
         -lepee \
         -lunbound \
         -leasylogging \
+        -lsodium \
```

Comment:
Since it's also required for android, could you please move that addition to line 292 ?